.github/workflows/ci.yaml

```diff
@@ -249,7 +249,62 @@ jobs:
         DOCKER_PASSWORD: ${{ secrets.QUAY_ROBOT_TOKEN }}
       run: |
         printf "$DOCKER_PASSWORD" | docker login --password-stdin --username "$DOCKER_USERNAME" quay.io
-    - run: make build-image BUILD_PLATFORMS=$BUILD_PLATFORMS PUSH_IMAGE=$PUSH_IMAGE TAG_IMAGE=true TARGET=$TARGET IMAGE_NAME=$IMAGE_NAME
+    - run: make build-image BUILD_ARCH=$BUILD_ARCH PUSH_IMAGE=$PUSH_IMAGE TARGET=$TARGET IMAGE_NAME=$IMAGE_NAME
+    - name: docker logout
+      if: ${{ always() }}
+      run: |
+        docker logout quay.io
+  authgear-image-arm64:
+    if: ${{ github.repository != 'oursky/authgear-server' }}
+    runs-on: ubuntu-24.04
+    needs: ["authgear-test", "authui-test"]
+    env:
+      TARGET: authgear
+      IMAGE_NAME: quay.io/theauthgear/authgear-server
+      PUSH_IMAGE: "${{ (github.repository == 'authgear/authgear-server' && github.event_name == 'push') && 'true' || 'false' }}"
```

Comment:
We should not push these `-arm64` or `-amd64` images. They are intermediate artifacts that should be combined and pushed at the end.

Reply:
I want to avoid the complication of using the digests by using a more understandable tag.
And in fact, the layers are always pushed. The difference is only whether that layer has a given tag name or not.
I think having a tag with a architecture suffix should not be a problem.